### ISSUE TITLE
enable/disable useTick test fails intermittently

### ISSUE THREAD

**Issue by Adamfsk** [inlet/react-pixi#118]
### Description
Just got this error when running the tests to publish a local version of the package. It succesfully built both before and after this failure, so seems to be an intermittent error. Since it's related to tick, it might be it's occasionally ticking more times than expected on occasion?
```
● stage › hook `useTick` › enable/disable useTick

    expect(received).toEqual(expected) // deep equality

    - Expected
    + Received

      Array [
        0,
        1,
    -   2,
      ]

      382 |       update(true)
      383 | 
    > 384 |       expect(fn.mock.calls.map(call => call[0])).toEqual([
          |                                                  ^
      385 |         0, // initial
      386 |         1,
      387 |         2
```

**Comment by inlet** [comment]
It depends on the react-reconciler package. These initial tests are written a while back, but it might be better to integrate https://github.com/testing-library/react-testing-library to make testing hooks more stable.

**Comment by inlet** [comment]
Fixed in #122
